build.gradle

```diff
@@ -380,9 +382,11 @@ publishing {
         if (project.hasProperty('DEPLOY_DIR')) {
             maven { url DEPLOY_DIR }
         }
+        mavenLocal()
     }
 }
 
 tasks.withType(JavaCompile).configureEach {
     options.encoding = 'UTF-8' // Use the UTF-8 charset for Java compilation
+    options.compilerArgs << "-Xmaxerrs" << "1000" // Show way more compiler errors
```

Comment:
This should be an IDE run configuration change, not a global change to the build.gradle.

Reply:
Just to clarify, are you referring to only a _local_ configuration change?

Reply:
Just to clarify, are you referring to only a _local_ configuration change?

Reply:
You can change your run configurations directly in IDEA settings. This is just the config that generates it.

That option does not need to be set when not porting.

Reply:
I changed it because I prefer compiling via Terminal instead of IDEA. But since we don't need that option anymore, I'll remove it.